src/v2/components/ChannelContents/mutations/moveConnectable.js

```diff
@@ -0,0 +1,24 @@
+import gql from 'graphql-tag'
```

Comment:
This file needs to be a `.ts` for type generation.

Reply:
Ah thx